### PR TITLE
readme: rewrite from org to markdown

### PR DESCRIPTION
**Pull request by imsofi**
Change reason: Markdown is a more common markup language compared to
emacs org mode. This will let more people be able to contribute back
into the documentation effort of `nix-minecraft`.

Also most editors do not have support for org-mode either, which further
complicates using it as a standard for the README of the project.

### CODE REVIEW

**Comment by imsofi** [comment]
This is also meant to be a first step to help the rewriting process of the README. Hopefully documenting all `TODO` sections, as well as building up the examples here inside the repo to help #6

File: README.md [hunk 1]
```diff
@@ -1,131 +1,128 @@
...
 }
-#+end_src
+```
 
 In your system configuration:
-#+begin_src nix
+```nix
 { inputs, ... }: # Make sure the flake inputs are in your system's config
 {
   imports = [ inputs.minecraft.nixosModules.minecraft-servers ];
```
Comment (Infinidoge) [inline README.md:24]:
```suggestion
  imports = [ inputs.nix-minecraft.nixosModules.minecraft-servers ];
```

This is actually an issue I haven't fixed in the main repo yet, apparently.

File: README.md [hunk 2]
```diff
@@ -134,15 +131,15 @@ For example,
...
 This option is quite powerful, and can be used for a number of things, though most notably it can be used for declaratively setting up mods or plugins for the server.
 
-This example takes an attrset of the IDs and hashes for Modrinth mods, fetches each one, and makes a folder containing those mods. (~linkFarmFromDrvs~ is quite useful because it can take a list of derivations and produce a folder suitable for this purpose.) The names in this attrset are meaningless, I only included them as convenient labels.
+This example takes an attrset of the IDs and hashes for Modrinth mods, fetches each one, and makes a folder containing those mods. (`linkFarmFromDrvs` is quite useful because it can take a list of derivations and produce a folder suitable for this purpose.) The names in this attrset are meaningless, I only included them as convenient labels.
 
-#+begin_src nix
+```nix
 {
   symlinks = {
     mods = pkgs.linkFarmFromDrvs "mods" (map pkgs.fetchModrinthMod (builtins.attrValues {
```
Comment (Infinidoge) [inline README.md:145]: (Right below this)

In these mod examples, `responseHash` should actually just be `hash`, another thing I forgot to fix in the current readme.

**Review by Infinidoge — APPROVED** [review]
Looks good to me, merging!

**Pull request merged**